Update Configuring custom images for the SKU Selector (#677)
vtexdocs/dev-portal-content: docs/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-configuring-custom-images-for-the-sku-selector.md

## Step by step

> 1. In the admin's Catalog, access **Products and SKUs**.

Reviewer: ```suggestion
1. Open the Admin and go to **Catalog > All Products**.
```

> 2. Select the desired product, click on the three dots button and on **View SKU list**.

Reviewer: ```suggestion
2. Find the desired product, and click the down arrow button. Then, click  **SKU**.
```

> 3. Select the desired SKU and click on the **Edit** button.
> 4. Click on the **Images** tab.
> 5. Upload the desired custom image, by clicking on **Insert**.

Reviewer: ```suggestion
3. Find the desired SKU and click the corresponding **Edit** button.
4. Click the **Images** tab.
5. Click the **Insert** button to add a new SKU image.
```

> 1. Access your store's Admin.

Reviewer: ```suggestion
1. Access the Admin.
```
